hs_discover/public/index.html

```diff
@@ -1,17 +1,218 @@
-{% extends "base.html" %}
-{% load pages_tags mezzanine_tags keyword_tags comment_tags hydroshare_tags %}
-{% block meta_title %}Sample{% endblock %}
+{% extends "newbase.html" %}
+<!--{% load pages_tags mezzanine_tags keyword_tags comment_tags hydroshare_tags %}-->
+{% block meta_title %}Discover{% endblock %}
 {% block main %}
-    <input type="hidden" id="static-url" value="{{ STATIC_URL }}">
-    <div class="container" id="discover-main">
-        <div class="row">
-            <div class="col-xs-12">
-                <h2 class="page-title">Success</h2>
-                <div>{{ sample_item }}</div>
-                <div id="app"></div>
+<!-- Google map needs explicit height or may not show up -->
+<!-- vue-date-pick needs font-size override or will inherit an odd oversized appearance from the import of the css -->
+<style type="text/css">
+    /* Always set the map height explicitly to define the size of the div
+     * element that contains the map. */
+    #map {
+        /* a height must be specified to make visible */
+        height: 50%;
+    }
+    .vdpHeadCellContent {
+        font-size: 1em;
+    }
+    .vdpCellContent {
+        font-size: 1em;
+    }
+    input {
+        /*padding-right: 30px;*/
+        font-size: 14px;
+    }
+    #map-view {
+        display: none;
+    }
+    #discover-footer {
+        /* align with filters and resource rows */
+        padding: 0;
+    }
+</style>
+<input type="hidden" id="qstring" value="{{ request.GET.q }}">
+<div class="container" id="discover-main">
```

Comment:
users can share search queries in emails/chat and post links that have a search term and even filters

Reply:
haven't we always had that?